static/less/fonts.less

```diff
@@ -16,18 +16,20 @@
   font-style: normal;
 }
 
-// IBM Plex
+// Roboto Mono
 
 @font-face {
-  font-family: 'IBM Plex';
-  src: url('../fonts/ibm-plex-mono-regular.woff2') format('woff2');
-  font-weight: normal;
+  font-family: 'Roboto Mono';
+  src: url('../fonts/roboto-mono-regular.woff2') format('woff2'),
+    url('../fonts/roboto-mono-regular.woff') format('woff');
+  font-weight: 400;
   font-style: normal;
 }
 
 @font-face {
-  font-family: 'IBM Plex';
-  src: url('../fonts/ibm-plex-mono-medium.woff2') format('woff2');
+  font-family: 'Roboto Mono';
+  src: url('../fonts/roboto-mono-medium.woff2') format('woff2'),
+    url('../fonts/roboto-mono-medium.woff') format('woff');
   font-weight: bold;
```

Comment:
The true weight of this font file is `500` (Medium), not `bold`/`700`. However, given that the current front-end code uses the `bold` keyword instead of `500`, we should define the font face as `bold` for now, pending further discussion about this in a future front-end TSC meeting.